README.md

```diff
@@ -131,6 +131,7 @@ In addition to embedding other Etch components, you can pass an arbitrary constr
 * Your function must be a constructor. It will be called with the `new` keyword and passed the JSX expression's properties and children as arguments.
 * The component object returned by your constructor must have an `element` field pointing to a DOM node. This element will be inserted into the DOM.
 * The component object returned by your constructor may have an `update` method. This method will be called whenever the same function appears in the same location in successive calls to `render`.
+* The component object returned by your constructor may have a `destroy` method. This method will be called when the component is removed from the DOM. If you do any event binding to the DOM in your component, this is where you should clean it up.
```

Comment:
We only need to clean up event handlers for DOM nodes that outlive the component, right? I wonder if we should say `"If you need to perform any cleanup associated with removing your component's element from the DOM, you can do it here"` I don't feel super strongly about this though. It might also be worth mentioning that you'll need to call the method manually when working with the root component?